tests/nlu/classifiers/test_diet_classifier.py

```diff
@@ -195,6 +195,7 @@ async def _train_persist_load_with_different_settings(
 
 
 @pytest.mark.skip_on_windows
+@pytest.mark.trains_model
```

Comment:
`test_process_gives_diagnostic_data` also trains, no?

Reply:
yeah, not sure why that one didn't get recorded...